### DISCUSSION TITLE
I want to read pages without interruptions and slowdowns

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#156] [Wants]
When I&#39;m a user, I just want to read the page, darnit! I don&#39;t want things popping in front of my face asking me to subscribe to notifications or email lists. Most importantly, I don&#39;t want to wait while ads load; or even worse, see the text all move around once the ads load. (And, while we&#39;re at it, let&#39;s disable all automatically playing videos and sounds.)

This, by the way, is why so many people use adblockers!

At this point, I think we need more strictly delineate pages for reading, versus web applications. There should be some friction before an application can consume lots of bandwidth, generate sound, nag me, etc.

<hr>

https://webwewant.fyi/wants/35/

**Comment by nikunjbhatt** [comment]
Yes, these things are very annoying. But what can be done to stop HTML popups (not the popup window/tab)? May be we need to develop HTML popup blockers like ad blockers!

The auto-playing videos are using a lot of bandwidth without consent. Browsers have decided to allow auto-playing videos and only muting them by default, this is stupidity. I have installed extension/plugin AutoplayStopper to stop auto-playing videos.
